libcontainer/factory_nabla.go

```diff
@@ -64,7 +67,28 @@ type NablaFactory struct {
 	Root string
 }
 
-// TODO(NABLA)
+func createRootfsISO(config *configs.Config) (string, error) {
+	rootfsPath := config.Rootfs
+	isoPath, err := storage.CreateIso(rootfsPath)
+	if err != nil {
+		// TODO: Add wrap
+		return "", err
+	}
+	targetISOPath := filepath.Join(rootfsPath, "rootfs.iso")
+
+	if err = utils.Copy(targetISOPath, isoPath); err != nil {
```

Comment:
Can `CreateIso` create `targetISOPath` instead? I think there was some reason for not to do that.

Reply:
I think this the problem in the original `runnc` was that the place where we wanted to put the ISO was in the `rootfs` directory, so it would be creating the ISO in the same folder it was creating an ISO of. 

Let me revisit this and see if it's possible to not have that cyclic file dependency.

Reply:
Addressed in b7e4d382f42657d2950ab630cb1d3582c0439571